.github/workflows/release.yml

```diff
@@ -1,13 +1,14 @@
-name: release
+name: Release CI
 
 # env:
 #   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 #   NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
 
 on:
   push:
-    branches:
-      - main
+    branches: [main, next, beta]
+  pull_request:
+    branches: [main, next, beta]
```

Comment:
Not sure why this was added - you wouldn't want to generate releases from pull-requests.

Reply:
Yes, I addressed the same concern, which should be fixed here https://github.com/storyblok/storyblok-js-client/pull/851